packages/query/src/block-processor.ts

```diff
@@ -261,7 +271,7 @@ export class BlockProcessor implements BlockProcessorInterface {
         // - detect LpNft position opens
         // - generate all possible position state metadata
         // - update idb
-        await this.identifyLpNftPositions(blockTx);
```

Comment:
can we update the description above `processTransactions(blockTx)` to accurately describe this function? Can we also move these existing comments to `identifyLpNftPositions()` and add comments to `identifyAuctionNfts()`.

Reply:
Whoops, good catch — fixed.